Gems/Camera/Code/Source/EditorCameraComponent.cpp

```diff
@@ -201,22 +240,42 @@ namespace Camera
         const CameraComponentConfig& config = m_controller.GetConfiguration();
         const float distance = config.m_farClipDistance * m_frustumViewPercentLength * 0.01f;
 
-        float tangent = static_cast<float>(tan(0.5f * AZ::DegToRad(config.m_fov)));
-        float height = distance * tangent;
-        float width = height * debugDisplay.GetAspectRatio();
+        float width;
+        float height;
+
+        if (config.m_orthographic)
+        {
+            width = config.m_orthographicHalfWidth;
+            height = width / debugDisplay.GetAspectRatio();
+        }
+        else
+        {
+            const float tangent = static_cast<float>(tan(0.5f * AZ::DegToRad(config.m_fov)));
+            height = distance * tangent;
+            width = height * debugDisplay.GetAspectRatio();
+        }
 
         AZ::Vector3 farPoints[4];
         farPoints[0] = AZ::Vector3( width, distance,  height);
         farPoints[1] = AZ::Vector3(-width, distance,  height);
         farPoints[2] = AZ::Vector3(-width, distance, -height);
         farPoints[3] = AZ::Vector3( width, distance, -height);
 
-        AZ::Vector3 start(0, 0, 0);
         AZ::Vector3 nearPoints[4];
-        nearPoints[0] = farPoints[0].GetNormalizedSafe() * config.m_nearClipDistance;
-        nearPoints[1] = farPoints[1].GetNormalizedSafe() * config.m_nearClipDistance;
-        nearPoints[2] = farPoints[2].GetNormalizedSafe() * config.m_nearClipDistance;
-        nearPoints[3] = farPoints[3].GetNormalizedSafe() * config.m_nearClipDistance;
+        if (config.m_orthographic)
```

Comment:
Adding a option to the camera component to choose orthographic camera vs perspective camera is a good approach, and for now it's great. For long term this approach should be run through UX team to gather feedback. Maybe they'd have arguments for a different approach, like for example providing two different camera components.

Reply:
Agreed! This is just a quick fix for now